markets/bfp-market/contracts/modules/LiquidationModule.sol

```diff
@@ -254,12 +253,18 @@ contract LiquidationModule is ILiquidationModule {
         liquidateCollateral(accountId, marketId, market, globalConfig);
 
         // Flag and emit event.
-        market.flaggedLiquidations[accountId] = msg.sender;
+        market.flaggedLiquidations[accountId] = ERC2771Context._msgSender();
```

Comment:
You're calling `_msgSender` 3 times. Store in a local var once then reuse the var. Depending on how the contracts were optimised, this could lead to 3 storage loads to check for `isTrustedMulticallForwarder` as well as unnecessary rechecks to verify and decode `msg.data` over and over and over.